### ISSUE TITLE
No GUI appears

### ISSUE THREAD

**Issue by ahernsean** [RamonUnch/AltDrag#36]
When I right-click on the taskbar icon and select About or Configure, no GUI appears. AltDrag appears to be functional, however, as windows do move when I alt-drag them.
![image](https://user-images.githubusercontent.com/1649300/110329174-457fa480-7fea-11eb-855f-850b2bc3b1de.png)

**Comment by RamonUnch** [comment]
Strange, did you try a restart of AltDrag? a full system restart?
Obviously I do not have this problem on my Win XP/8/10 machines, so if you could try to see if an other application is interfering or a specific setting from AltDrag or Windows is responsible, this would permit me to reproduce the problem and to fix it.

Are you using the latest 1.39? did the problem occur with older versions? eg 1.37 or 1.35?
If needed the ini file can be manually edited to change the config. (middle click on the tray icon).

**Comment by RamonUnch** [comment]
Maybe if you just updated, post your altdrag.ini file here and try to use the one in the release (extract from the zip). Maybe this is a configuration upgrade problem. in this case If I have the problematic ini file I *should* be able to fix it.

**Comment by ahernsean** [comment]
Well, I can no longer reproduce the problem.

I did try restarting AltDrag several times. The problem even persisted after a reboot. And I had this happen on the prior version of AltDrag too.

I did the following after your suggestions:
1. Quit AltDrag
2. Move aside my .ini file by renaming it.
3. Start AltDrag. At this point, it all works fine. About shows up. The config panel shows up.
4. Quit AltDrag
5. Move my .ini file by renaming back to its original AltDrag.ini
6. Start AltDrag. Normal behavior from here on out.
At this point, AltDrag is working perfectly. I'm seeing all of the expected GUI items. Dunno what happened there, but there's nothing to explore if it's no longer happening. Thanks for your time.
